.dockerignore

```
@@ -0,0 +1,17 @@
+__pycache__
```

Worth noting that this probably isn't have much effect - `.dockerignore` has a frustrating subtlety in that it's _slightly_ different to `.gitignore`; this _only_ ignores files/folders with this name that exist at the _root level_ of the repo. It won't ignore any in the `todo_app` folder for example. We probably want:
```suggestion
**/__pycache__
```

([example comparison](https://zzz.buzz/2018/05/23/differences-of-rules-between-gitignore-and-dockerignore/))